public/speed-testJS.js

```diff
@@ -584,8 +584,11 @@
                 myChart.setOption(option, true);
         }
         var baseUrl = (version === 'IPv6') ? 'http://' + testPlan.baseUrlIPv6 : 'http://' + testPlan.baseUrlIPv4;
-
-        var uploadHttpConcurrentTestSuite = new window.uploadHttpConcurrentProgress(baseUrl + '/upload', 'POST', 1, 15000, 15000, 2, uploadHttpOnComplete, uploadHttpOnProgress,
+        var concurrentRuns = 1;
```

Comment:
Here and the 2 lines below, instead of hardcoding values here, maybe track them using a variable/const/config value.